### ISSUE TITLE
Updating overwrites docker-credential-ecr-login symlink with out-of-date version

### ISSUE THREAD

**Issue by jobimrobinsantos-drizly** [docker/for-mac#6180]
Summary: installing an update to Docker Desktop for Mac overwrote the symlink `/usr/local/bin/docker-credential-ecr-login`, pointing it at `/Applications/Docker.app/Contents/Resources/bin/docker-credential-ecr-login`, **which is out of date** (v0.5.0). I have installed the latest release (v0.6.0) with homebrew (`brew install docker-credential-helper-ecr`), which was the target of the symlink before the Docker Desktop update was installed.

### Expected behavior
```
$ docker pull ${ECR_URI}
# (image is pulled)

$ docker-credential-ecr-login -v
amazon-ecr-credential-helper
Version:    0.6.0
Git commit: 69c85dc
```

### Actual behavior
```
$ docker pull ${ECR_URI}
Error response from daemon: Head ${ECR_URI}: no basic auth credentials

$ docker-credential-ecr-login -v
amazon-ecr-credential-helper
Version:    0.5.0
Git commit: b19192b6522b2da02d14ec394c331f3b1a70efe2
```

### Information
  - macOS Version: 11.6.3
  - Intel chip or Apple chip: Intel Core i7
  - Docker Desktop Version: 4.1.1 (69879)

#### Is it reproducible?
Yes, this has happened to me twice now. The first time I thought it was a fluke. Now I believe it is due to how updates are installed.

#### Is the problem new?
v0.6.0 was released not very long ago, so it's difficult to say if this issue was always around and only now surfaced.

#### Did the problem appear with an update?
Yes.

**Comment by StefanScherer** [comment]
@jobimrobinsantos-drizly We have released Docker Desktop 4.5.0 which contains version 0.6.0 of docker-credential-ecr-login.

**Comment by jobimrobinsantos-drizly** [comment]
Thank you! 🙏
